server/crawler/router.ts

```diff
@@ -9,10 +9,24 @@ router.get('/', async (req, res, next) => {
     const { url } = req.query
     if (!url) {
       res.status(400).send('MISSING_ASSET_ID')
-      return 
+      return
     }
-    const data = await getCralwerData(url as string)
-      res.send(data)
+    const data = await crawlData(url as string)
+    res.send(data)
+  } catch (error) {
+    next(error)
+  }
+})
+
+router.get('/image', async (req, res, next) => {
+  try {
+    const { url } = req.query
+    if (!url) {
+      res.status(400).send('MISSING_IMAGE_URL')
+      return
+    }
+    const data = await crawlImage(url as string)
```

Comment:
Now this api allow getting any image from any url
I think it would be better to limit the api by using same url used in `/`, fetch if there is an ogImage, then return the ogImage if exists.
Also we should use stream and pipe for image data api